src/main.rs

```diff
@@ -109,10 +109,6 @@ const DURATION_DAY: Duration = Duration::from_secs(60 * 60 * 24);
 /// by panicking with this type instead of a string.
 struct ExitPanic(i32);
 
-/// Similar to the above, but allows us to exec a new command
-/// as our final act.
-struct ExecPanic(std::process::Command);
```

Comment:
🎊 this is good motivation for me to remove the last of my error handling sins and remove the Last Remaining ExitPanic.